readme.md

```diff
@@ -0,0 +1,265 @@
+<a name="readme-top"></a>
+
+<!--
+HOW TO USE:
+This is an example of how you may give instructions on setting up your project locally.
+
+Modify this file to match your project and remove sections that don't apply.
```

Comment:
You did a good job implementing the README file :+1: but as a [project requirement](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/hello_microverse_project.md) you should customize it because it would make your project looks more professional.